influxdb2/webif/templates/index.html

```diff
@@ -148,44 +109,4 @@
 		</tbody>
 	</table>
 
-</div>
 {% endblock bodytab1 %}
-
```

Comment:
Warum hast Du die Blöcke für die noch kommenden Tabs entfernt (Das Plugin ist als noch in develop gekennzeichnet)? Um sie nicht anzuzeigen hätte es gereicht den Tabcount auf 1 zu setzen (was Du ja auch gemacht hast).